src/ontoweaver/transformer.py

```diff
@@ -70,24 +72,19 @@ def __call__(self, row, i):
             str: Each split item from the cell value.
         """
         for key in self.columns:
-            if self.valid(row[key]):
-                assert(type(row[key]) == str)
-                items = row[key].split(self.separator)
-                for item in items:
-                    if self.valid(item):
-                        yield str(item)
-                    else:
-                        logging.warning(
-                            f"Encountered invalid content when splitting values of column column: `{key}`, line: {i}, in `split` transformer. Skipping cell value: `{item}`")
-            else:
-                logging.warning(f"Encountered invalid content when mapping column: `{key}`, line: {i}, in `split` transformer. Skipping cell value: `{row[key]}`")
-
+            items = str(row[key]).split(self.separator)
+            for item in items:
+                gen =  self.create(item)
+                try:
+                    yield next(gen)
+                except StopIteration:
+                    pass
```

Comment:
It seems that there has been a struggle with generators here.
The difficulty probably stems from the implementation of `create` as a generator (see comment above).
With `create` as a simple function, this should be as simple as:
```suggestion
            for item in items:
                yield self.create(item)
```